data-model/SDP-sample-records/script_sample1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/script/1",
+  "label": "Serto (script)",
+  "associated_writing_systems": ["Syriac"],
+  "temporal_coverage": {
+    "date_human-readable": "post-7th c. - present",
+    "date_normalized": "0650/"
```

Comment:
Is there an ISO standard way to indicate 'to the present'?

Also, This raises a difficult question for something like Estrangela which fell off but never fully disappeared around the 8th century, and had a resurgence in the tenth century. I don't think simple 'from' and 'to' ranges will be sufficient?